packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -3,38 +3,44 @@ title: Set up VA.gov locally
 ---
 
 # Set up VA.gov locally
+These instructions cover the installation of dependencies needed for running VA.gov locally.
+- **Node setup** - steps for installing `node` which is needed for building, testing, and running the local development server
+- **Get the source code** - steps for cloning the various repos needed to run VA.gov
+- **Start up the front end** - installing and running VA.gov locally
+- **Backend and local tools setup** - links to other setup instructions for running `vets-api` and retrieving static content
 
-## Node setup
+## Prerequisites
+- [`homebrew`](http://brew.sh/) is recommended for for installing `nvm` but other installation approaches are on [`nvm` Github page](https://github.com/creationix/nvm#installation-and-update)
```

Comment:
Explicitly call out prerequisites. We may want to remove this and just give people a command to install homebrew.